util/blockdevice/util.go

```diff
@@ -24,6 +24,62 @@ import (
 	"mayadata.io/cstorpoolauto/unstruct"
 )
 
+/*
+Find sample bd is here -
```

Comment:
Can we actually use this sample bd against our UT?

Reply:
I will do it in some other PR.
Added this as a reference as we are getting values of the nested map. It will help to review and write new support also.